istioctl/pkg/install/pre-check.go

```diff
@@ -44,7 +44,9 @@ import (
 )
 
 const (
-	minK8SVersion = "1.13"
+	// Minimum K8 version required to run latest version of Istio
+	// https://preliminary.istio.io/docs/setup/platform-setup/
```

Comment:
link to the preliminary site, not the istio.io one

Reply:
Thanks